src/TodoItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function TodoItem(props){
+    const {date} = props;
+    const done = props.isDone;
+    let style = '';
+    (!done) ? style = 'none' : style = 'line-through';
```

Comment:
Не пиши такое никогда. Это очень плохо читается.
В eslint есть прекрасное правило, которое запрещает это. https://eslint.org/docs/rules/no-unused-expressions
Тем более можно сделать лучше.

```suggestion
    style = done ? 'line-through' : 'none';
```